apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -534,6 +539,12 @@ const Ticket = new GQLListSchema('Ticket', {
             defaultValue: false,
             isRequired: true,
         },
+        customOrder: {
```

Comment:
It’s for kanbad board style columns. We don’t have this style at the moment. But we want it in a future

Reply:
so.. lets add kanbanOrder